src/spam-protection/spam-protection-request-sender.ts

```diff
@@ -0,0 +1,26 @@
+import { RequestSender, Response } from '@bigcommerce/request-sender';
+
+import { Checkout } from '../checkout';
+import { ContentType, RequestOptions } from '../common/http-request';
+
+const DEFAULT_PARAMS = {
+    include: [
+        'cart.lineItems.physicalItems.options',
+        'cart.lineItems.digitalItems.options',
+        'customer',
+        'promotions.banners',
+    ].join(','),
+};
+
+export default class SpamProtectionRequestSender {
+    constructor(
+        private _requestSender: RequestSender
+    ) {}
+
+    validate(checkoutId: string, token: string, { timeout }: RequestOptions = {}): Promise<Response<Checkout>> {
+        const url = `/api/storefront/checkouts/${checkoutId}/spam-protection`;
```

Comment:
Does the endpoint need to return a response body? Is it because the checkout object changes its state (i.e.: has an additional flag) after validation? Or could it just return a status code? If you do want to send back the checkout object, perhaps you don't need to ask for the additional includes?

Reply:
Yup, the value for `shouldRunSpamProtectionCheck` will change in the checkout object after verification. I removed the includes.